src/app/shared/courses/courses.service.ts

```diff
@@ -13,19 +13,25 @@ export class CoursesService {
         return this.courses.slice(0, page * displayLimit);
     }
 
+    getCoursesLength(): number {
```

Comment:
this must be private and marked as a temporary solution (`with a TODO`)

Reply:
I am using this method in form-course component too